examples/electron_phonon/NPB.xyz

```diff
@@ -0,0 +1,80 @@
+78
```

Comment:
@baumeier could you please also add some tests for the new functionality? see for instance the [dftgwbse test](https://github.com/votca/PyVOTCA/blob/master/tests/test_dftgwbse.py)